File: Cosmos-Android/app/src/main/java/wannabit/io/cosmostaion/model/type/IrisVote.java
package wannabit.io.cosmostaion.model.type;

import com.google.gson.annotations.SerializedName;

public class IrisVote {

    @SerializedName("voter")
    public String voter;

    @SerializedName("proposal_id")
    public String proposal_id;

    @SerializedName("option")
    public String option;
}

File: Cosmos-IOS/Cosmostation/Cosmostation/Model/IrisVote.swift
//
//  IrisVote.swift
//  Cosmostation
//
//  Created by yongjoo on 10/12/2019.
//  Copyright © 2019 wannabit. All rights reserved.
//

import Foundation


public class IrisVote {
    
    static let OPTION_YES           = "Yes"
    static let OPTION_NO            = "No"
    static let OPTION_VETO          = "NoWithVeto"
    static let OPTION_ABSTAIN       = "Abstain"
    
    var voter: String = ""
    var proposal_id: String = ""
    var option: String = ""
    
    init(_ dictionary: [String: Any]) {
        self.voter = dictionary["voter"] as? String ?? ""
        self.proposal_id = dictionary["proposal_id"] as? String ?? ""
        self.option = dictionary["option"] as? String ?? ""
    }
}
